docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -113,13 +118,15 @@ devel                                   In development (2.11 unreleased, trunk)
 Development and stable version maintenance workflow
 -----------------------------------------------------
 
-The Ansible community develops and maintains Ansible and ``ansible-base`` on GitHub_.
+The Ansible community develops and maintains Ansible and ``ansible-core`` on GitHub_.
 
 Collection updates (new modules, plugins, features and bugfixes) will always be integrated in what will become the next version of Ansible. This work is tracked within the individual collection repositories.
 
-Ansible and ``ansible-base`` provide bugfixes and security improvements for the most recent major release. The previous
-major release of ``ansible-base`` will only receive fixes for security issues and critical bugs.``ansible-base`` only applies
```

Comment:
This also had a problem with no whitespace after "bugs." so the following inline highlighting didn't work.

Reply:
ah, thanks for fixing that